File: accommodation_service_queries/src/main/java/pl/edu/pg/accommodation/model/Room.java
package pl.edu.pg.accommodation.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.extern.jackson.Jacksonized;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Room {
    private long id;
    private String name;
    private int capacity;
    private String features;
    private float price;
}

File: tour_operator/hotels/Room.py
class Room:
    def __init__(self, room):
        self.capacity = room["capacity"]
        self.name = room["name"]
        self.features = room["features"]

        self.numberOfRooms = None
        if "numberOfRooms" in room:
            self.numberOfRooms = room["numberOfRooms"]

        self.price = None
        if "price" in room:
            self.price = room["price"]
